project/ProjectPlugin.scala

```diff
@@ -25,6 +25,7 @@ object ProjectPlugin extends AutoPlugin {
       val scalatestplusScheck: String = "3.1.0.0-RC2"
       val skeuomorph: String          = "0.0.23"
       val slf4j: String               = "1.7.30"
+      val http4s: String              = "0.21.1"
```

Comment:
```suggestion
      val http4s: String              = "0.21.4"
```

Reply:
I'll put all the changes in the same commit